Common/Core/EnumExtensions.cs

```diff
@@ -25,22 +21,17 @@ public static class EnumExtensions
             string description = x.GetDescription();
          * 
          */
-        public static string GetDescription(this Enum value)
+        public static string? GetDescription(this Enum value)
         {
             Type type = value.GetType();
-            string name = Enum.GetName(type, value);
+            var name = Enum.GetName(type, value);
```

Comment:
Why is "var" better than "string"? Don't see a reason that getName could return something else then string - maybe just a question of style preference?

Reply:
var doesn't mean it's variable :-)
They should probably have come up with another token as it sound weird to javascript devs, but var in c# means "let the compiler deal with the type as it's pretty obvious".
And this is how I use it: GetName obviously returns a string, so no need to declare the variable as such.

Reply:
> it sound weird to javascript devs

I'm a Java dev, not javascript - now I'm offended 😄 But anyway, it sounds weird for a Java dev as well. In javascript there is a similar concept but without type checking at all, unless one uses TypeScript.

Reply:
I wasn't saying this for you ;-) In Java, var has been introduced with Lombok and also serves the exact same purpose (being lazy about mentioning the type, and wanting to keep only useful information).